cmd/kube-controller-manager/app/core.go

```diff
@@ -538,6 +538,9 @@ func startGarbageCollectorController(ctx ControllerContext) (controller.Interfac
 	discoveryClient := ctx.ClientBuilder.DiscoveryClientOrDie("generic-garbage-collector")
 
 	config := ctx.ClientBuilder.ConfigOrDie("generic-garbage-collector")
+	// The garbage collector controller requires 2 API calls for each object deletion: GET and DELETE.
+	// As usually GET calls are much cheaper than DELETE ones, increase the available QPS to allow more DELETEs per second.
```

Comment:
This reasoning doesn't make sense to me, I think it's trying to say it's OK to double it because GETs are nearly free? I would rather just say the GC needs more throughput, I think this comment will confuse people. If anything I'd say we can't make it unlimited until APF propagates the expense of watch notifications back to deletes.

Reply:
Yeah, it doesn't make much sense when I'm reading it now. I changed as suggested, I hope it is more readable now.